src/client/services/server-api/index.js

```diff
@@ -26,7 +26,7 @@ const ServerAPI = {
 
   querySearch(query){
     const queryClone=_.assign({},query);
-    if(/(uniprot:\w+|ncbi:[0-9]+|hgnc:\w+)$/.test(queryClone.q)){queryClone.q=queryClone.q.split(':')[1];}
+    queryClone.q=queryClone.q.replace(/(uniprot|ncbi|hgnc)\s*:/gi,"").replace(/\s\s+/g, ' ').replace(/\s+$/, '');
```

Comment:
Can we trigger the replace only when there is a single token ? Maybe:

`q=queryClone.q.replace(/^(uniprot|ncbi|hgnc):\w+$/i,"")...`